typescript/infra/scripts/utils.ts

```diff
@@ -155,7 +155,19 @@ export function getCoreVerificationDirectory(environment: DeployEnvironment) {
 }
 
 export function getCoreRustDirectory(environment: DeployEnvironment) {
-  return path.join('../../', 'rust', 'config', environment);
+  let envstr;
+  switch (environment) {
```

Comment:
You can use this: https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/f5d1fbbd03f9cc59841a581eb1829a28c253c766/typescript/infra/src/config/environment.ts#L46

Reply:
No longer relevant with removing the env from the path.